net/safe-search/Makefile

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2018 Gregory L. Dietsche <Gregory.Dietsche@cuw.edu>
+# This is free software, licensed under the MIT License
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=safe-search
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Gregory L. Dietsche <Gregory.Dietsche@cuw.edu>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/safe-search
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=Safe Search
+	PKGARCH:=all
+endef
+
+define Package/safe-search/description
+This package prevents adult content from appearing in search results by
+configuring dnsmasq to force all devices on your network to use Google and
+Bing's Safe Search IP addresses. This is designed to be approperiate for most
+businesses and families. The default filtering rules do not interfere with
+normal web browsing.
+endef
+
+define Package/safe-search/conffiles
+/etc/config/safe-search
+endef
+
+define Build/Compile
+endef
+
+define Package/safe-search/install
+	$(INSTALL_DIR) $(1)/etc/uci-defaults
+	$(CP) ./files/safe-search.defaults $(1)/etc/uci-defaults/safe-search
+
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_CONF) ./files/safe-search.conf $(1)/etc/config/safe-search
+
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) ./files/safe-search-update $(1)/usr/sbin/safe-search-update
+
+	$(INSTALL_DIR) $(1)/etc/safe-search/enabled
+	$(INSTALL_DIR) $(1)/etc/safe-search/available
+	$(INSTALL_DATA) ./files/hosts/* $(1)/etc/safe-search/available/
+
+	chmod -R ugo+rX $(1)/etc/safe-search/
```

Comment:
`INSTALL_DIR` should already have the correct permission bits set (`install -d -m0755`), so this line is not necessary?

Reply:
you are correct, thanks!